infrastructure/pcluster-manager.yaml

```diff
@@ -25,6 +26,35 @@ Parameters:
     Type: String
     Default: 3.1.2
 
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Authentication
+        Parameters:
+          - EnableAuth
+          - EnableMFA
+      - Label: 
+          default: Initial Admin User
+        Parameters: 
+          - AdminUserEmail
+          - AdminUserPhone
+      - Label:
+          default: ParallelCluster API
+        Parameters:
+          - Version
+          - PublicEcrImageUri
+    ParameterLabels:
+      EnableMFA:
+        default: Require Multi-Factor Authentication for all Users
+      EnableAuth:
+        default: Enable Auth Roles (e.g., Admin, User, Guest)
```

Comment:
This is actually:

```
When false this enables users to create an account & login. Only use this value for tutorials or short lived deployments.
```

Reply:
Ok, I'll update. Is this long form the Description or you want it to be the Label?